index.js

```diff
@@ -338,7 +338,7 @@ async function fastifyStatic (fastify, opts) {
           file = file
             .replace(rootPath.replace(/\\/g, '/'), '')
             .replace(/^\//, '')
-          const route = encodeURI(prefix + file).replace(/\/\//g, '/')
+          const route = (prefix + file).replace(/\/\//g, '/')
```

Comment:
Not needed anymore?

Reply:
You can follow the discussion here
https://github.com/delvedor/find-my-way/pull/253#issuecomment-1118516930

Reply:
Ah brill, thanks @climba03003